.travis.yml

```diff
@@ -8,13 +9,13 @@ sudo: false
 
 before_install:
   - go get golang.org/x/tools/cmd/vet
-  - go get github.com/golang/lint/golint
+  - go version | grep -qE '(go1.5|go1.6)' && go get github.com/golang/lint/golint
```

Comment:
@vbatts what about doing something like what I did for containerd to check for a minimum version?: https://github.com/docker/containerd/pull/132/commits/c6680da78b7777fc1c814c5f35ee166a83e5408a